databases/postgresql-cloudnativepg/README.md

```diff
@@ -0,0 +1,7 @@
+# Deploying a PostgreSQL cluster with CLOUDNATIVEPG operator on GKE
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/kubernetes-engine-samples&cloudshell_tutorial=cloudshell/tutorial.md&cloudshell_workspace=gke-postgresql-cloudnativepg)
+
+This example shows how to build and deploy a PostgreSQL cluster using [Kubernetes](https://kubernetes.io) and Day 1 & 2 operations such as Monitoring with [Cloud Monitor](https://cloud.google.com/monitoring) and disaster recovery with [Backup for GKE](https://cloud.google.com/kubernetes-engine/docs/add-on/backup-for-gke/concepts/backup-for-gke).
+
+Follow this tutorial at [https://cloud.google.com/kubernetes-engine/docs/tutorials/database/cloudnativepg](https://cloud.google.com/kubernetes-engine/docs/tutorials/database/cloudnativepg)
```

Comment:
Do we know if that's going to be the URL for sure? If we're not certain I'm happy if we remove that line and we can merge it, and then fix this README afterwards once the tutorial is published

Reply:
THe current path for the in-review file is: https://cloud.google.com/kubernetes-engine/docs/tutorials/stateful-workloads/cloudnativepg

It is possible that it'll change

Reply:
URL changed